torchlog/torchlog.go

```diff
@@ -41,7 +41,7 @@ func init() {
 func getPrefix(level string, color *color.Color) string {
 	currentTime := time.Now().Format("15:04:05")
 	toColoredString := color.SprintFunc()
-	return toColoredString(fmt.Sprintf("%s[%s]", level, currentTime))
+	return toColoredString(fmt.Sprintf("%s[%s] ", level, currentTime))
```

Comment:
Yup, agree it's better. Wasn't sure whether you meant to include it here or
another diff. This is good though.
On Fri, Jun 3, 2016 at 10:17 AM Prashant Varanasi <notifications@github.com>
wrote:

> In torchlog/torchlog.go
> <https://github.com/uber/go-torch/pull/44#discussion_r65713885>:
>
> > @@ -41,7 +41,7 @@ func init() {
> >  func getPrefix(level string, color *color.Color) string {
> >  	currentTime := time.Now().Format("15:04:05")
> >  	toColoredString := color.SprintFunc()
> > -	return toColoredString(fmt.Sprintf("%s[%s]", level, currentTime))
> > +	return toColoredString(fmt.Sprintf("%s[%s] ", level, currentTime))
>
> Minor formatting change to make the output a little nicer. The previous
> output looked like:
>
> INFO[19:00:29]Run pprof command: go tool pprof -raw -seconds 30 main.test prof.cpu
> INFO[19:00:29]Writing svg to torch.svg
>
> I thought adding a space would make the output a little more readable:
>
> INFO[19:00:29] Run pprof command: go tool pprof -raw -seconds 30 main.test prof.cpu
> INFO[19:00:29] Writing svg to torch.svg
>
> Caught this while updating the examples in README
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/uber/go-torch/pull/44/files/0a522f28435d4715d4eb4aca8d7f9f0fc71e290e#r65713885>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAF7P63XMASsMH-L4Y5YhmWfXvFdd9IMks5qIDdygaJpZM4ItPAO>
> .
>